### PR TITLE
added background, and worked on making the cover page

### PR DESCRIPTION
**Pull request by CodeSmitty**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <meta charset="UTF-8">
   <title>Emily Bennett Portfolio</title>
   <link rel="stylesheet" href="./style.css">
-
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@900&display=swap" rel="stylesheet">
```
Comment (christinameador) [inline index.html:10]: Do we need this import statement? The CSS file has two import statements for the two fonts used.

Comment (CodeSmitty) [reply index.html:10]: I did not add the imports in the css. I used the link import in the html to get as close to the font from the design img.

**Review by christinameador — APPROVED** [review]
I like the overall direction, just confused about the import statement in the HTML file. Otherwise, looks really good!

File: index.html (hunk 1 above)
Comment (christinameador) [reply index.html:10]: Right, I had the two fonts imported in the CSS file. Should I change them to the HTML file?

Comment (CodeSmitty) [reply index.html:10]: I can remove it, if you prefer.

**Pull request merged**